implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs

```diff
@@ -22,7 +24,7 @@ pub struct CreateCommand {
 
     /// Authorized Identifiers of secure channel initiators
     #[arg(short, long, value_name = "IDENTIFIER")]
-    authorized_identifier: Option<Vec<IdentityIdentifier>>,
+    authorized_identifiers: Option<Vec<IdentityIdentifier>>,
```

Comment:
Good catch! Could you also change `value_name = "IDENTIFIER"` to `value_name = "IDENTIFIERS"`?

Reply:
updated